exercises/crypto-square/src/lib.rs

```diff
@@ -1,3 +1,3 @@
-pub fn encrypt(_: &str) -> String {
-    unimplemented!()
+pub fn encrypt(input: &str) -> String {
```

Comment:
I'm OK with this being `input`.

Another possibility I would accept is `plaintext`.